Libraries/LibWeb/MediaCapabilitiesAPI/MediaCapabilities.h

```diff
@@ -1,15 +1,104 @@
 /*
  * Copyright (c) 2024, Jamie Mansfield <jmansfield@cadixdev.org>
+ * Copyright (c) 2025, Psychpsyo <psychpsyo@gmail.com>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #pragma once
 
 #include <LibWeb/Bindings/PlatformObject.h>
+#include <LibWeb/EncryptedMediaExtensions/EncryptedMediaExtensions.h>
 
 namespace Web::MediaCapabilitiesAPI {
 
+// https://w3c.github.io/media-capabilities/#dictdef-videoconfiguration
+struct VideoConfiguration {
+    String content_type;
+    unsigned long width;
+    unsigned long height;
+    Optional<unsigned long> bitrate;
+    double framerate;
+    Optional<bool> has_alpha_channel;
+    Bindings::HdrMetadataType hdr_metadata_type;
+    Bindings::ColorGamut color_gamut;
+    Bindings::TransferFunction transfer_function;
+    String scalability_mode;
+    Optional<bool> spatial_scalability;
+
+    // https://w3c.github.io/media-capabilities/#valid-video-configuration
+    bool is_valid_video_configuration() const;
+};
+
+// https://w3c.github.io/media-capabilities/#dictdef-audioconfiguration
+struct AudioConfiguration {
+    String content_type;
+    Optional<String> channels;
+    Optional<unsigned long> bitrate;
+    Optional<unsigned int> samplerate;
+    Optional<bool> spatial_rendering;
+
+    // https://w3c.github.io/media-capabilities/#valid-audio-configuration
+    bool is_valid_audio_configuration() const;
+};
+
+// https://w3c.github.io/media-capabilities/#mediaconfiguration
+struct MediaConfiguration {
+    Optional<VideoConfiguration> video;
+    Optional<AudioConfiguration> audio;
+
+    // https://w3c.github.io/media-capabilities/#valid-mediaconfiguration
+    bool is_valid_media_configuration() const;
+};
+
+// https://w3c.github.io/media-capabilities/#keysystemtrackconfiguration
+struct KeySystemTrackConfiguration {
+    String robustness = ""_string;
```

Comment:
```suggestion
    String robustness;
```

Already default initializes to empty string, here an below.

Reply:
Done.